Week1/assignments/Bright/Exercise#12.js

```diff
@@ -0,0 +1,31 @@
+ function palindrome(str) {
+
+    var lent = str.length;
+    var mid = Math.floor(lent/2);   //Round lent downward to its nearest integer:
+
+        // 
+    for ( var i = 0; i < mid; i++ ) {
+        if (str[i] !== str[lent - 1 - i]) {
+            return false;
+        }
+    }
+
+    return true;
+}
+// test case
+console.log(palindrome("mom")); 
+
+
+
+
+
+/* function reverseString(str) {
```

Comment:
Try to not submit code with `dead code` included. Please take this out.
Good attempt though :-)

Reply:
alright thanks

Reply:
Do i create a new pull request after editing ?